### PR TITLE
Vxl vnl math testing

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

**Comment by hjmjohnson** [comment]
@thewtex @dzenanz Trying to address annoying failures on the ITK dashboard.

**Comment by hjmjohnson** [comment]
https://open.cdash.org/testSummary.php?project=2&name=vnl_test_math&date=2018-11-15

File: core/vnl/tests/test_math.cxx [hunk 1]
```diff
@@ -5,6 +5,9 @@
 #include <vnl/vnl_complex.h> // for vnl_math::abs(std::complex)
```
Comment (dzenanz) [inline core/vnl/tests/test_math.cxx]: In commit message, "Numerial" -> "Numerical"

File: core/vnl/tests/test_math.cxx [hunk 2]
```diff
@@ -627,7 +627,7 @@ static void test_math()
 #define RETURN_TYPE_TEST( funcname, argtypename, returntypename ) \
   { \
   const bool test_return_type = std::is_same< decltype( vnl_math :: funcname( static_cast< argtypename >(123.4) ) ), returntypename >(); \
-  TEST("vnl_math::" #funcname " returns " #returntypename " type", test_return_type, true); \
+  TEST("vnl_math::" #funcname "<" #argtypename "> returns " #returntypename " type", test_return_type, true); \
```
Comment (dzenanz) [inline core/vnl/tests/test_math.cxx]: I hope this macro syntax compiles 😄

File: core/vnl/vnl_math.h [hunk 3]
```diff
@@ -137,7 +137,6 @@ namespace vnl_math
```
Comment (dzenanz) [inline core/vnl/vnl_math.h]: In commit message: remoe -> remove.

**Comment by thewtex** [comment]
Thanks for helping keep the dashboard clean @hjmjohnson :+1:

**Pull request merged**